src/python/WMCore/WMSpec/Steps/Executors/CMSSW.py

```diff
@@ -95,6 +97,8 @@ def execute(self, emulator=None):
         logging.info('User files are %s', userFiles)
         logging.info('User sandboxes are %s', userTarball)
 
+        scramArch = self.getSingleScramArch(scramArch)
```

Comment:
We could raise an exception, but honestly, if this was a problem then we'd have seen this problem at least once in the last 5 years :)
Though I think it would be good to have a logging line before the cmssw bootstrap (L121) just printing the scram command + release + arch